docs/developer/PROW_CI.md

```diff
@@ -64,12 +64,278 @@ The jobs run can be seen in PRs and in the links in the README.md file of OADP r
 
 OADP operator master branch is tested against the last 3 minor OCP releases. To Update an OCP version the project is tested against, see [Update OCP version](#update-ocp-version).
 
-### TODO Creating new release branch
+### Creating new release branches
 
-https://github.com/openshift/oadp-operator/issues/1227
+To create new OADP release branches (they must follow the pattern `oadp-major.minor`, [example](https://github.com/openshift/oadp-operator/tree/oadp-1.4)):
+- First, create CI files for each new repository branch.
+- Create the release branches in each one of related repositories of OADP, and OADP operator repository itself.
+- The new release branches must be updated:
+  - Downstream repositories release branches may not need updates (except OADP operator, which always need updates).
+  - Upstream repositories release branches need to be rebased.
+
+> **Note**: Try to always create release branches from default branches (usually called master).
+
+> **Note**: Documentation should live only in default branches. For example, for OADP operator, `docs/` and `blogs/` folders can be deleted in release branches.
```

Comment:
`can` or `should` be deleted? Leaving option to leave it would be softer then explicitly stating that we only want the default branch to be updated with docs. Does the default branch carries over old design documentation, or proper instructions for a given branch? What if the instructions are changing in master branch? Do we keep the correct ones for a given branch?

Reply:
we currently do not delete, I want to start doing this

we have old docs in master yes, like this https://github.com/openshift/oadp-operator/blob/master/docs/upgrade_1-1_to_1-2.md

Reply:
So I would make this document clear that we do delete and then continue with implementation (deletion). Note, this won't affect rebases and cherry-picks?